stable/pgadmin/values.yaml

```diff
@@ -22,6 +22,26 @@ strategy: {}
   #   maxSurge: 0
   #   maxUnavailable: 1
 
+## Server definitions will be loaded at launch time. This allows connection
+## information to be pre-loaded into the instance of pgAdmin in the container.
+## Ref: https://www.pgadmin.org/docs/pgadmin4/4.13/import_export_servers.html
+##
+serverDefinitions:
+  ## If true, server definitions will be created
+  ##
+  enabled: false
+
+  servers: |-
+  #  "1": {
```

Comment:
does it make sense to have this as yaml instead, and use toJson?

Reply:
I thought about that, but it feels easier this way for people that already had pgadmin servers,json files to simply copy it over. Also, the documentation of pgadmin now applies more or less.